### PR TITLE
frontend first commit

### PR DESCRIPTION
**Pull request by BingBlog**
The frontend framework

### CODE REVIEW

File: visualdl/frontend/build.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+wget -r -nH --level=0 --cut-dirs=7 getprod@product.scm.baidu.com:/data/prod-64/baidu/third-party/nodejs/nodejs_6-10-0-1_PD_BL/  --user getprod --password getprod --preserve-permissions
+
+cd output/
+tar zxf nodejs.tar.gz
+export PATH=`pwd`/bin:$PATH
+
+cd ../../
+
+# 使用 内部源 安装依赖包
```
Comment (Superjomn) [inline visualdl/frontend/build.sh:17]: remove all the chinese comments

File: visualdl/frontend/mock/example/mock.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * 获取 mock 响应数据
+ *
+ * @param {string} path 请求路径名
+ * @param {Object} queryParam 查询参数信息
+ * @param {Object} postParam post 的查询参数信息
+ * @return {Object}
+ */
+module.exports = function (path, queryParam, postParam) {
+    return {
+        // 可以通过该属性来设置响应的延时，也可以设为值为'0,100'，表示随机 0-100ms 的延时，默认 0
```
Comment (Superjomn) [inline visualdl/frontend/mock/example/mock.js:18]: remove these

Comment (deqingli) [reply visualdl/frontend/mock/example/mock.js:18]: we will remove it when build

File: visualdl/frontend/BCLOUD [hunk 3]
```diff
@@ -0,0 +1 @@
+BUILD_SUBMITTER -x -m baidu/jady/san-searchbi-static -c "cd baidu/jady/san-searchbi-static && sh build.sh" -u ./
```
Comment (jacquesqiao) [inline visualdl/frontend/BCLOUD]: rm this file

**Comment by jacquesqiao** [comment]
visualdl/frontend/package-lock.json 这个文件好长，都是我们需要的么

**Review by deqingli — CHANGES_REQUESTED** [review]
this pr will be blocked, please add the english annotation and pre-commit

**Pull request closed without merging**